agave_app/python/pyrenderer.h

```diff
@@ -104,6 +104,8 @@ class OffscreenRenderer : public RenderInterface
   virtual int SetBoundingBoxColor(float, float, float);
   virtual int ShowBoundingBox(int32_t);
   virtual int ShowScaleBar(int32_t);
+  virtual int SetFlipAxis(int32_t, int32_t, int32_t);
```

Comment:
is this another adding something that should have been there? I thought the flip axis was an feature you already added

Reply:
yes, this was just added for completeness because I noticed it was missing.  Shouldn't even affect any runtime behaviors.